projects/library/chip/src/chip-collection.component.ts

```diff
@@ -167,32 +144,33 @@ export class TsChipCollectionComponent implements OnInit, AfterViewInit, AfterCo
   public keyManager!: FocusKeyManager<TsChipComponent>;
 
   /**
-   * Function when touched
+   * Manage selections
    *
    * @internal
    */
-  public onTouched = () => { };
+  public selectionModel!: SelectionModel<TsChipComponent>;
 
   /**
-   * Function when changed
+   * Function when touched
    *
    * @internal
    */
-  public onChange: (value: string[]) => void = () => { };
+  public onTouched = () => { };
 
   /**
-   * Manage selections
+   * Function when changed
    *
    * @internal
    */
-  public selectionModel!: SelectionModel<TsChipComponent>;
+  public onChange: (value: string[]) => void = () => { };
```

Comment:
Probably from trying to move the `member-order` rule over which requires properties before methods etc. I ended up having issues with it since it didn't like our private property to be with the getter/setter which we do in _every file_ so I had to pull the rule.